### ISSUE TITLE
.tif file does not render 2d Z-stack however will render 3d image of the 2d Z-stack. This is an arch linux install

### ISSUE THREAD

**Issue by ghost** [janelia-flyem/NeuTu#397]
Hello all,

First a quick note: I put green over the majority of the neuron just to make sure I'm not accidentally sharing an image I'm not supposed to share, I left the axon so you can see the image quality of the 3d image.

So Neutu is rendering the 3D image of confocal Z-stack data as expected. But for some reason the 2d-images will not show up. The format is .tif and is less than 1GB. Any guidance on what to read or perhaps what code output to give you will be appreciated. Thank you for this nice program.

I load this image through file -> open to which the left part of the screen, with the blank window where the 2d images should appear. I press the 3D button and in very little time the 3d image shows up with good quality and loads very fast. 

![neutu-render-issue](https://user-images.githubusercontent.com/71096974/214603081-d70c8cf5-71c3-4e0d-b451-2e0c12a32341.png)

**Comment by ghost** [comment]
This may be due to the program being built for EM data. I don't know how this program works 'under the hood' so feel free to reopen if this should not be an issue with 2D rendering.

**Comment by olbris** [comment]
You are correct, our primary use is EM, and we don't have experience displaying confocal images with NeuTu.

Unfortunately, "The format is .tif" is a less informative statement than one might think. TIFF is more of a meta-format, and there are still any number of ways an image can be partially incompatible with any given TIFF reader. This goes double for images produced by microscopes, which can be very idiosyncratic. 

If you can post a log file it would help. It should be located in your home directory, `.neutube.z` folder, named `log.txt`. No guarantees we can do anything, as EM is our focus, but we can take a look.
